portfolio/src/main/webapp/pagination.js

```diff
@@ -82,10 +94,11 @@ class CommentButton {
       this.hideCommentAtIndex(this.currentCommentsPageIndex);
     } else {
       this.hideCommentAtIndex(this.currentCommentsPageIndex);
-      this.requestAndDisplayNewCommentsPage(this.cursorString);
+      await this.requestAndDisplayNewCommentsPage(this.cursorString);
```

Comment:
haha! this is a good solution, but deserves a comment. 

// wait until the new page is loaded. we are disallowing pagination while the comments are loaded.